### PR TITLE
fix(conditional styling): conditional visibility for images and alpha-numerics in display layouts

### LINKED ISSUE #7718: [Conditional Styling] Visibility option on Conditional Styling is broken
#### Summary

When you conditionally style widgets or images  in a Display Layout, the option to toggle the visibility is broken. 

https://github.com/nasa/openmct/assets/48999852/7848fd86-2090-4206-ab60-aefa26ba2b27

#### Expected vs Current Behavior

The eye toggle in Conditional Styles allows for images to appear or disappear based on the condition. 

#### Steps to Reproduce

1. Download this JSON file that includes an example of the bug. [DL .json](https://github.com/nasa/openmct/files/15242291/DL.json)
2. This JSON has the following: a Display Layout that shows a Condition Set output, the telemetry's current value, and an image that uses the Condition Set. This image has the conditional styling logic such that when the Condition Set outputs the string "invisible", the image disappears. When the Condition Set outputs the string "visible", the image appears.
3. Observe that when the Condition Set outputs "Invisible", the NASA logo does not disappear as it should. 

#### Environment

* Open MCT Version:  4.0.0 next
* Deployment Type:  All 
* OS:
* Browser: Chrome

#### Impact Check List

- [x] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### LINKED ISSUE #7782: Alphanumerics cannot be conditionally hidden
#### Summary

Condition visibility does not work with alpha-numeric display elements.

#### Expected vs Current Behavior
Display builders should be able to hide or show alphanumeric elements based on the output of a condition set.

#### Steps to Reproduce
1. Import [the attached display layout]([Alphanumeric visibility.json](https://github.com/user-attachments/files/16241562/Alphanumeric.visibility.json)).
2. Switch to real-time mode.
3. Observe that the SWG alpha-numeric element is always visible
4. Click on the SWG alpha-numeric and observe that it has conditional styling applied that should render it invisible when the value of the SWG drops below 0.

#### Impact Check List

- [x] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [ ] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [ ] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by ozyx**
Closes #7718
Closes #7782 

### Describe your changes:

Conditionally shows image views and alpha-numeric views within display layouts when visibility is toggled via a conditional style. This is achieved via Vue's `v-show` conditional rendering, which applies a `display: none;` when the `v-show` condition is false.

Adds a couple regression tests plus a test which generates a localstorage state with a basic condition set that can be used in other e2e tests to save execution time.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?

### CODE REVIEW

**Review by scottbell — CHANGES_REQUESTED** [review]
Looking good, but I had trouble with the testing instructions in #7782 unless I added some optional chaining [here](https://github.com/nasa/openmct/blob/mct7718-conditional-styling-visibility-option-on-conditional-styling-is-broken/src/plugins/displayLayout/components/TelemetryView.vue#L49) and [here](https://github.com/nasa/openmct/blob/mct7718-conditional-styling-visibility-option-on-conditional-styling-is-broken/src/plugins/displayLayout/components/TelemetryView.vue#L51) as it appeared the domain object was null:
```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')
    at eval (TelemetryView.vue:49:75)
```

After the optional chaining changes, I could test without issue.

https://github.com/user-attachments/assets/6a0a7b6f-baca-412e-a13f-ea3fbce37139

File: e2e/tests/functional/plugins/styling/conditional/displayLayoutConditionalStyling.e2e.spec.js [hunk 1]
```diff
@@ -28,8 +28,8 @@ import {
 } from '../../../../../appActions.js';
 import { expect, test } from '../../../../../pluginFixtures.js';
 
-const TINY_IMAGE_BASE64 =
-  'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAIAQMAAAD+wSzIAAAABlBMVEX///+/v7+jQ3Y5AAAADklEQVQI12P4AIX8EAgALgAD/aNpbtEAAAAASUVORK5CYII';
+const RICK_JPG =
```
[the remark this answers is not in the document]
Comment (ozyx) [reply e2e/tests/functional/plugins/styling/conditional/displayLayoutConditionalStyling.e2e.spec.js:31]: it's not, but you bring up a good point. I'm thinking i'll go with the base64 image instead.

[the remark this answers is not in the document]
Comment (scottbell) [reply e2e/tests/functional/plugins/styling/conditional/displayLayoutConditionalStyling.e2e.spec.js:31]: If there's already precedent, I'm fine either way.

**Review by scottbell — APPROVED** [review]
Works great - well done!

**Comment by ozyx** [comment]
Waiting to merge this due due an anomaly i found in the visual tests related to me changing a div to a button element. investigating

**Pull request merged**